evergreen/evergreen.yml

```diff
@@ -465,6 +615,23 @@ tasks:
       commands:
         - func: run-plain-auth-tests
 
+    - name: aws-auth-test
+      depends_on:
+        - variant: windows-64-compile
+          name: compile
+      commands:
+        - func: bootstrap-mongo-orchestration
+          vars:
+            AUTH: "auth"
+            ORCHESTRATION_FILE: "auth-aws.json"
+            TOPOLOGY: "server"
+        - func: add-aws-auth-variables-to-file
+        - func: run-aws-auth-test-with-regular-aws-credentials
+        - func: run-aws-auth-test-with-aws-EC2-credentials
+        - func: run-aws-auth-test-with-assume-role-credentials
+        - func: run-aws-auth-test-with-aws-credentials-as-environment-variables
+        - func: run-aws-auth-test-with-aws-credentials-and-session-token-as-environment-variables
+
```

Comment:
Are you not testing on the ECS container?

Reply:
That's correct. There was a conversation in slack AWS chat regarding ECS testing on windows. As a summary: .NET driver should run ECS tests manually (which I do) until Linux testing Epic is complete. The major issue here is that ECS test is set up to use Ubuntu machines, while .NET driver is not yet ready for Linux testing.